ansible/.env.j2

```diff
@@ -0,0 +1,17 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+
+# Details for Trello API Credentials:
+TRELLO_API_KEY={{trello_api_key}}
+TRELLO_API_TOKEN={{trello_api_token}}
+TRELLO_BOARD_ID=656851f21489d3cebc979ded
+TRELLO_TODO_LIST_ID=656851f20f4a7c4638899308
+TRELLO_ACTIVE_LIST_ID=656851f238936766c85715d9
+TRELLO_DONE_LIST_ID=656851f2348b0a7bcf295353
```

Comment:
Having these hardcoded is absolutely fine for security purposes, but it does keep us currently quite tied into a specific environment. Not something I'd actually worry about here, but on a real project I'd pre-emptively pull these out into variables, but set the current values as a "default" argument in your playbook so you don't need to paste them in every time!